clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -46,9 +46,11 @@
 
     public static final String SEND_BUFFER_CONFIG = "send.buffer.bytes";
     public static final String SEND_BUFFER_DOC = "The size of the TCP send buffer (SO_SNDBUF) to use when sending data. If the value is -1, the OS default will be used.";
+    public static final int SEND_BUFFER_LOWER_BOUND = -1;
```

Comment:
@guozhangwang @hachikuji I think we can do this without a KIP, but I would like to very with you.

Reply:
sounds good to me.